src/Sep.Test/SepWriterHeaderTest.cs

```diff
@@ -162,6 +162,25 @@ public void SepWriterHeaderTest_Add_Twice_Throws()
         var e = Assert.ThrowsException<ArgumentException>(() => header.Add("A"));
         Assert.AreEqual("Column name 'A' already exists (Parameter 'colName')", e.Message);
     }
+    
+    [TestMethod]
+    public void SepWriterHeaderTest_Try_Add_Once_Is_True()
```

Comment:
Try_Add => TryAdd, same below, please add missing test for when header already written, and see CI

thank you for the PR. could you elaborate on why exactly you do this in this way? Would a Contains method better solve your need?